package.json

```diff
@@ -31,6 +33,8 @@
     "@typescript-eslint/eslint-plugin": "^5.30.4",
     "@typescript-eslint/parser": "^5.30.4",
     "eslint": "^8.19.0",
+    "jest": "^28.1.3",
```

Comment:
https://vitest.dev/ :trollface:

Reply:
Ciekawa alternatywa, ale jak sami twierdzą to nie jest jeszcze `ready for production` ;)

Reply:
u mnie działa :trollface: